### ISSUE TITLE
java.io.IOException: File x.pdf seems to be non-pdf - but x.pdf is fine

### ISSUE THREAD

**Issue by sedimentation-fault** [Yanpas/PdfMerger#2]
[sriperumbudur12.pdf](https://github.com/Yanpas/PdfMerger/files/2461559/sriperumbudur12.pdf)
[sriperumbudur12Supple.pdf](https://github.com/Yanpas/PdfMerger/files/2461560/sriperumbudur12Supple.pdf)

Trying to merge the two attached PDFs with

`java -jar pdfmerger.jar sriperumbudur12.pdf sriperumbudur12Supple.pdf out.pdf`

I get:

```
Processing sriperumbudur12.pdf
Processing sriperumbudur12Supple.pdf
File "sriperumbudur12Supple.pdf" seems to be non-pdf
java.io.IOException: File "sriperumbudur12Supple.pdf" seems to be non-pdf
        at yanpas.pdfmerger.Merger.appendDoc(Merger.java:42)
        at yanpas.pdfmerger.Merger.addDocument(Merger.java:69)
        at yanpas.pdfmerger.PdfMerger.main(PdfMerger.java:33)
```

but _sriperumbudur12Supple.pdf_ is a valid PDF file, as you can assure yourself by opening it in any viewer. At least my Acrobat Professional does not complain at all and displays it without problems.

Any hints?

**Comment by sedimentation-fault** [comment]
O.K., it turns out that, even though _sriperumbudur12Supple.pdf_ is displayed correctly, it is NOT a valid PDF after all - and _PdfMerger_ is right in coughing at it:

Opening _sriperumbudur12Supple.pdf_ in _PdfXChange_, I get a warning saying that some object in the PDF has the wrong orientation. _PdfXChange_ could correct the error. The new PDF could be processed by _PdfMerger_ without problems.

I attach the new, corrected PDF.
[sriperumbudur12Supple.pdf](https://github.com/Yanpas/PdfMerger/files/2461684/sriperumbudur12Supple.pdf)

**Comment by Yanpas** [comment]
I've released [new version 1.1](https://github.com/Yanpas/PdfMerger/releases/tag/1.1) where bumped version of pdfbox library. That seems to fixed your issue.

Thanks!
